accounts/ethkey/cli/src/main.rs

```diff
@@ -267,16 +317,13 @@ fn execute<S, I>(command: I) -> Result<String, Error> where I: IntoIterator<Item
 		let (phrase, keypair) = in_threads(move || {
 			let mut it = brain_recover::PhrasesIterator::from_known_phrase(&known_phrase, BRAIN_WORDS);
 			move || {
-				let mut i = 0;
-				while let Some(phrase) = it.next() {
-					i += 1;
-
+				for (i, phrase) in (&mut it).enumerate() {
 					let keypair = Brain::new(phrase.clone()).generate().unwrap();
```

Comment:
Out of scope: we could replace the unwrap() with expect here

Reply:
Before the change the loop effectively started with 1 because of how increment was structured. I made the change to balance the fix in the beginning of the scope.

Reply:
done